apidef/api_definitions.go

```diff
@@ -842,24 +841,53 @@ type ClientAuthData struct {
 	ClientSecret string `bson:"client_secret" json:"client_secret"`
 }
 
+// HeaderData holds the configuration for the custom header to be used for OAuth authentication.
+type HeaderData struct {
```

Comment:
`HeaderData` is not used anymore, I guess we can remove it.

Reply:
done